src/plugins/intel_gpu/include/intel_gpu/plugin/program.hpp

```diff
@@ -195,7 +195,8 @@ void CreateUnaryEltwiseOp(Program& p, const std::shared_ptr<ngraph::Node>& node,
 void CreateElementwiseOp(Program& p,
                          const std::shared_ptr<ngraph::Node>& node,
                          cldnn::eltwise_mode mode,
-                         std::vector<float> coefficients = {});
+                         std::vector<float> coefficients = {},
+                         bool pythondiv = false);
```

Comment:
The default value for `m_pythondiv` is `true`.
https://docs.openvino.ai/latest/openvino_docs_ops_arithmetic_Divide_1.html

Reply:
done